tests/bootstrap.php

```diff
@@ -0,0 +1,31 @@
+<?php
```

Comment:
Tests file can be ignored for now, it should be added to phpcs ruleset since the repo doesn't have one, @DevikVekariya can you open up an issue to add PHPCS to the project?

Reply:
Yes @thrijith I am opening one for phpcs pipeline